src/inputs/CheckmarksMultipleSelect/CheckmarksMultipleSelect.js

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React, { useEffect, useState } from 'react';
+import {
+  FormControl,
+  Select,
+  Divider,
+  Checkbox,
+  Button,
+  makeStyles,
+  FormControlLabel,
+  FormGroup,
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import DeleteForeverIcon from '@material-ui/icons/DeleteForever';
+
+const useStyles = makeStyles((theme) => ({
+  multipleSelect: {
+    width: '100%',
+  },
+  removeButton: {
+    color: theme.palette.error.main,
+    textTransform: 'none',
+    fontSize: theme.typography.body1.fontSize,
+    display: 'flex',
+    justifyContent: 'left',
+  },
+  menuList: {
+    paddingRight: 0,
+  },
+  menuItem: {
+    marginLeft: 0,
+    marginRight: 0,
+  },
```

Comment:
Just a note to remember to talk about how you created the list, it seems either I made a non compliant mockup or you didn't know how to use MUI according to my mockup.